content/manager_architecture/security.md

```diff
@@ -99,6 +99,34 @@ As part of the agent's installation script, Cloudify's internal CA certificate i
 propagated to the agent's host in order to validate the manager's certificate.
 There are no agent-host certificates.
 
+#### Customizing SSL for internal communication
+
+It is possible to override the internal Manager certificate, and the CA certificate
+dugin bootstrap. In order to provide a custom internal CA certificate (which will be
+used by the agents), the `ca_certificate` and optionally `ca_key` must be set.
```

Comment:
Let's put a link here to bootstrap inputs, just so that people are clear what "set" means.